packages/grafana-ui/src/components/Layout/Space.story.tsx

```diff
@@ -0,0 +1,64 @@
+import { StoryFn, Meta } from '@storybook/react';
+import React from 'react';
+
+import { Box } from './Box/Box';
+import { Space } from './Space';
+import mdx from './Space.mdx';
+
+const meta: Meta<typeof Space> = {
+  title: 'General/Layout/Space',
+  component: Space,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export default meta;
+
+export const Horizontal: StoryFn<typeof Space> = (args) => {
+  return (
+    <div style={{ display: 'flex' }}>
+      <Box borderStyle={'solid'} padding={1}>
+        Box without space
+      </Box>
+      <Box borderStyle={'solid'} padding={1}>
+        Box with space on the right
+      </Box>
+      <Space {...args} />
+      <Box borderStyle={'solid'} padding={1}>
+        Box without space
+      </Box>
+    </div>
+  );
+};
+
+Horizontal.args = {
+  v: 0,
```

Comment:
Could we add some argTypes for these? You can use helpers like the ones in [Box](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Layout/Box/Box.story.tsx#L62). Then it makes it easier to see what options you can select.